src/main/scala/org/apache/mesos/chronos/scheduler/jobs/constraints/LikeConstraint.scala

```diff
@@ -0,0 +1,24 @@
+package org.apache.mesos.chronos.scheduler.jobs.constraints
+
+import org.apache.mesos.Protos
+
+case class LikeConstraint(attribute: String, value: String) extends Constraint {
+
+  val regex = value.r
+
+  def matches(attributes: Seq[Protos.Attribute]): Boolean = {
+    attributes.foreach { a =>
+      if (a.getName == attribute) {
+        a.getText.getValue match {
+          case regex(_*) => return true
```

Comment:
Scala's internal regex is great for finding regexp groups etc.
For a simple matching filter, please use the simple form: `regex.pattern.matcher(value).matches()`